source/query-bar.txt

```diff
@@ -69,7 +69,7 @@ as in the following filter example:
    { "_id": { "$oid": "58eee4f43d1a10a4e5339aac" } }
 
 When you have entered a valid MongoDB query, the :guilabel:`Filter`
-label is no longer red and :guilabel:`Apply` button is enabled. 
+label is no longer red and :guilabel:`Find` button is enabled. 
```

Comment:
See previous comment. Only using the full element name because this feature is a little tricky the first time that you use it.